test/integration/fixtures/blocks/core__social-links.serialized.html

```diff
@@ -1,3 +1,3 @@
-<!-- wp:social-links -->
+<!-- wp:social-links {"layout":{"type":"flex"}} -->
```

Comment:
I was not expected this change since it's the default value? Do we know why it gets included in the serialization?

Reply:
You're right! I think it's good now as before I was adding the above attribute even if it didn't had any `justify` css classes in the migration function.

Thanks for catching this as it confused me a bit as it was a default from a `supports` flag, but it is used in `edit` function, so it makes sense that this shouldn't be there.